docs/conf/atlantic/2024/sponsors/prospectus.rst

```diff
@@ -0,0 +1,275 @@
+:template: {{year}}/generic.html
```

Comment:
@ericholscher have we gone through this and adjusted anything? Do you think we should?

Reply:
This is still using the old format. I will update it using our latest from Portland.

Reply:
Updated 👍